go/codebuild/README.md

```diff
@@ -0,0 +1,75 @@
+# AWS SDK for Go code examples for AWS CodeBuild
+
+## Purpose
+
+These examples demonstrate how to perform several CodeBuild operations.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the AWS SDK for Go Developer Guide.
+
+## Running the code
+
+### BuildProject/BuildProject.go
+
+This example builds a CodeBuild project.
+
+`go run BuildProject -p PROJECT`
+
+- _PROJECT_ is the name of the CodeBuild project.
+
+The unit test mocks the service client and the `StartBuild` function.
+
+### ListBuilds/ListBuilds.go
+
+This example lists your CodeBuild builds.
+
+`go run ListBuilds.go`
+
+### ListProjects/ListProjects.go
+
+This example lists your CodeBuild projects.
+
+`go run ListProjects.go`
+
+### Notes
+
+- We recommend that you grant this code least privilege,
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
```

Comment:
*AWS Identity and Access Management User Guide*.

Reply:
Fixed